.storybook/main.js

```diff
@@ -2,9 +2,10 @@ const path = require('path');
 const sassFunctions = require('sass-functions');
 const sass = require('sass');
 
-const ROOT_PKG_PATH = path.join(__dirname, '../../../');
+const ROOT_PKG_PATH = path.join(__dirname, '../');
```

Comment:
will we run storybook from root or from packages? ⚖️  about the same?

Reply:
Almost yes but I think it would make a difference in load times. SSR-preview is already pretty slow because it has to load all stories of ONE package but we load all stories in all packages it could get worse.

I think it's ok for subpackages to have their own `start`, `lint`, `build` scripts